spec.proto

```diff
@@ -269,19 +269,23 @@ message ProtoExecutionResult {
     string errorMessage = 3;
     /// Stacktrace of the error
     string stackTrace = 4;
-    /// Byte array containing screenshot taken at the time of failure.
-    repeated bytes screenShot = 5;
+    /// [DEPRECATED, use failedScreenshot] Bytes containing screenshot taken at the time of failure.
+    bytes screenShot = 5 [deprecated=true];
+    /// Bytes containing screenshot taken at the time of failure.
+    bytes failedScreenshot = 6;
     /// Holds the time taken for executing this scenario.
-    int64 executionTime = 6;
+    int64 executionTime = 7;
```

Comment:
the field numbers should not be changed in protobuf. Refer https://developers.google.com/protocol-buffers/docs/proto3